includes/admin/class-admin.php

```diff
@@ -1162,6 +1173,33 @@ private function check_deprecation_warnings() {
             }
         }
 
+		/**
+		 * Check old inline fontawesome setting.
+		 * If it was enabled by default, we show an admin notice.
+		 * On dismiss we keep the setting.
+		 *
+		 * @since x.x
+		 */
+		private function check_font_awesome_setting() {
+			if ( ! current_user_can( 'administrator' ) ) {
+				return;
+			}
+
+			if ( ! WPBDP_App_Helper::is_bd_page() ) {
+				return;
+			}
+
+			if ( ! wpbdp_get_option( 'enqueue-fontawesome-styles', false ) ) {
+				return;
+			}
+
+			$this->messages[] = array(
+				str_replace( array( '[', ']' ), array( '<a href="https://wordpress.org/plugins/font-awesome" target="_blank" rel="noopener nofollow">', '</a>' ), _x( 'We changed how FontAwesome is integrated in Business Directory. Go [here] to install the officle "FontAwesome styles" plugin.', 'admin', 'business-directory-plugin' ) ),
```

Comment:
I know str_replace is used throughout this plugin for translations like this, but it's not the right way. It should use sprintf

Reply:
There's a minor typo here, the word `official` is accidentally spelled `officle`.